Makefile

```diff
@@ -28,6 +28,7 @@ LD := $(PREFIX)ld
 # note: the makefile must be set up so MODERNCC is never called
 # if MODERN=0
 MODERNCC := $(PREFIX)gcc
+PATH_MODERNCC := PATH=$(TOOLCHAIN)/bin:PATH $(MODERNCC)
```

Comment:
Why not just `MODERNCC := PATH=$(TOOLCHAIN)/bin:PATH $(PREFIX)gcc`?

Reply:
That could work as well but maybe some future hypothetical `MODERNCC` might want the variable alone or something (I don't really have a good reason here).